sale_disable_inventory_check/tests/test_disable_inventory_check.py

```diff
@@ -8,11 +8,29 @@ class TestDisableInventoryCheck(common.TransactionCase):
     def setUp(self):
         super(TestDisableInventoryCheck, self).setUp()
 
+        self.category_1 = self.env['product.category'].create({
+            'name': 'Category 1',
+            'property_is_check_stock': True,
+        })
+        self.category_2 = self.env['product.category'].create({
+            'name': 'Category 2',
+            'property_is_check_stock': False,
+        })
+
         # Create stockable product
         self.product_1 = self.env['product.product'].create({
             'name': 'Product 1',
             'type': 'product',
             'uom_id': self.env.ref('product.product_uom_unit').id,
+            'property_is_check_stock': True,
+            'categ_id': self.category_1.id
+        })
+        self.product_2 = self.env['product.product'].create({
+            'name': 'Product 2',
+            'type': 'product',
+            'uom_id': self.env.ref('product.product_uom_unit').id,
```

Comment:
@hieulucky111 if you need to direct id then use `self.ref('product.product_uom_unit')`